test/test.js

```diff
@@ -78,258 +84,209 @@ describe('YeldaChat', () => {
       }
 
       yeldaChat.init(mockData)
-      expect(yeldaChat.webChatContainer).not.to.be.null
+      setTimeout(() => {
+        expect(yeldaChat.webChatContainer).not.to.be.null
+      })
     })
   })
 
   describe('yeldaChat.setupChat', () => {
-
     describe('yeldaChat.setupChat initial checks', () => {
-      it('should return null if assistantId is missing', () => {
+      it('should return null if assistantId is missing', async () => {
         // Reset the DOM
         yeldaChat.unLoadChat()
         const mockData = {
           'assistantSlug': 'testClient',
           'chatPath': 'chat'
         }
 
-        const response = yeldaChat.setupChat(mockData)
-
-        assert.typeOf(response, 'null', 'setupChat function returns null')
+        await yeldaChat.setupChat(mockData)
         expect(yeldaChat.webChatContainer).to.be.null
       })
-      it('should return null if assistantSlug is missing', () => {
+
+      it('should return null if assistantSlug is missing', async () => {
```

Comment:
the test labels should be updated to match the real tests : it's not it should return null,, but webChatContainer should be null 
and getAssistantSettings should return null